src/System.Globalization.Calendars/tests/CalendarHelpers.cs

```diff
@@ -377,8 +378,15 @@ public static void TwoDigitYearMax_Invalid(Calendar calendar)
         [MemberData(nameof(DateTime_TestData))]
         public static void GetEra_Invalid(Calendar calendar, DateTime dt)
         {
-            // TODO: This fails for HebrewCalendar, TaiwanLunisolarCalendar and JapaneseLunisolarCalendar
-            // Assert.Throws<ArgumentOutOfRangeException>(() => calendar.GetEra(dt));
+        	// JapaneseCalendar throws on Unix (ICU), but not on Windows
+            if ((calendar is JapaneseCalendar && PlatformDetection.IsWindows) || calendar is HebrewCalendar || calendar is TaiwanLunisolarCalendar || calendar is JapaneseLunisolarCalendar)
```

Comment:
I have no idea why some calendars throw and others don't. Seems like the only thing these have in common (apart from Japanese) is that they are lunisolar. That said, other lunisolar calendars do throw